### PR TITLE
Update oncogenicity logic and VUS logic

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java [hunk 1]
```diff
@@ -31,6 +31,19 @@ public void testProcessQuery() throws Exception {
         indicatorQueryResp = IndicatorUtils.processQuery(query, null, null, null, true);
         assertEquals("The oncogenicity is not matched in variant summary.", "The KRAS V14I mutation is known to be oncogenic.", indicatorQueryResp.getVariantSummary());
 
+        // Check critical case
+        query = new Query(null, null, null, "BRAF", "V600E", null, "Melanoma", null, null, null);
+        indicatorQueryResp = IndicatorUtils.processQuery(query, null, null, null, true);
+        assertTrue("The geneExist in the response should be set to true", indicatorQueryResp.getGeneExist() == true);
```
Comment (JiaoJiao123) [inline core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java]: do you want to replace "indicatorQueryResp.getGeneExist() == true" with "indicatorQueryResp.getGeneExist()"?
Same for other similar assertTrue statement.

Comment (JiaoJiao123) [inline core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java]: do you want to replace "indicatorQueryResp.getGeneExist() == true" with "indicatorQueryResp.getGeneExist()"?
Same for other similar assertTrue statement.

Comment (zhx828) [inline core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java]: I think in this way the logic is relatively more clear. Let's keep in this way for now. I will check some examples to see what convention people usually use.

Comment (JiaoJiao123) [inline core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java]: ok sure.

**Pull request merged**